app/views/foreman/job_templates/install_errata_-_katello_ansible_default.erb

```diff
@@ -20,6 +20,34 @@ kind: job_template
 <%= render_template('Run Command - Ansible Default', :command => "zypper -n install -t patch #{advisories}") %>
 <% else -%>
 <% advisories = input(:errata).split(',').map { |e| "--advisory=#{e}" }.join(' ') -%>
-<%= render_template('Run Command - Ansible Default', :command => "yum -y update-minimal #{advisories}") %>
+---
+- hosts: all
+  tasks:
+    - name: Collect bootc status
+      shell:
+        cmd: 'bootc status --json'
+      register: bootc_status
+      ignore_errors: true
+    - name: Parse bootc status json
+      set_fact:
+        bootc_status_json: "{{ bootc_status.stdout | from_json }}"
+      when: bootc_status.rc == 0
+    - name: Enable bootc overlay
+      shell:
+        cmd: 'bootc usr-overlay'
+      register: out
+      ignore_errors: true
+      when: bootc_status_json is defined and bootc_status_json['status']['booted'] != 'null'
+    - debug: var=out
+    - name: Install errata via dnf for image mode machines
+      shell:
+        cmd: 'dnf -y --transient update-minimal <%= advisories %>'
```

Comment:
Do we still need --transient if we are already enabling usr-overlay before for image mode hosts?

Reply:
That's a great point! I can take out the usr-overlay parts.

Reply:
Yeah, I'm not super clear on why usr-overlay is needed at all.. Is --transient not always available?

Reply:
--transient is not possible when the Ansible package module is used. Some of our playbooks use the package module, which I didn't want to change. Eventually the package module should support the use of transient, which we'll implement eventually.

I took at look at what it would take to remove the use of package module, but in foreman_ansible we rely on a state input... messing with that sounds risky.